source/docs/yearly-changes/yearly-changelog.rst

```diff
@@ -1,240 +1,177 @@
-
-.. important:: MCAP Export requires Pro Licensing
-
-.. note:: Documentation on configuring and extracting logs will be available soon.
-
-.. grid:: 2
+Additional Swerve Language Support
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-   .. grid-item-card:: Log Extractor
+.. image:: images/tuner-multi-language.png
+   :alt: Multiple language support in Tuner
+   :width: 550
 
-      Logs can be extracted and converted to compatible formats directly in Tuner X.
+The implementation of the Swerve API has been moved to C++ to improve portability and performance. With this change, we are announcing Swerve Generator and Swerve API support for Python and C++.
 
-      .. image:: images/tuner-x-log-extractor.png
-         :alt: Log extractor page in Tuner X
+These changes result in a substantial performance improvement, as the odometry thread no longer pauses during Java garbage collection using native Phoenix swerve request calls. Custom swerve request calls are still available; however, their usage may not benefit from the performance improvements due to GC overhead.
 
-   .. grid-item-card:: Foxglove Log Analysis
+.. note: The previous Java swerve implementation is still available with the prefix ``Legacy`` added to it.
 
-      Logs can then be analyzed in `Foxglove <https://foxglove.dev/>`__ to identify hardware failures, tuning gains, etc.
+Improved Swerve Requests
+^^^^^^^^^^^^^^^^^^^^^^^^
 
-      .. image:: images/foxglove-example.png
-         :alt: Picture of foxglove analyzing data
+After further testing, the SysId swerve requests have been improved to output usable gains for both drive and steer motors, as well as the ``HeadingController`` used in ``FieldCentricFacingAngle``.
 
-Signal API Improvements
-^^^^^^^^^^^^^^^^^^^^^^^
+Additionally, ``ApplyChassisSpeeds`` and ``SwerveModule.apply()`` now optionally accept (robot-relative) wheel force feedforward vectors. This allows the robot to more closely follow acceleration along autonomous paths.
 
-Users can now disable signals by setting their update rate to 0 Hz. To reduce boilerplate when disabling signals, we have added an ``optimizeBusUtilization()`` function on device objects. This will automatically disable all signals that have not explicitly been given an update frequency with ``setUpdateFrequency()``. There is also a ``ParentDevice.optimizeBusUtilizationForAll()`` static function that takes a list of devices to optimize. Additionally, update frequencies are automatically reapplied when devices reset.
+Java Units Support
```

Comment:
Add a simple code snippet of using Java units with StatusSignal, and maybe control/config